extensions/ql-vscode/src/query-history.ts

```diff
@@ -277,10 +301,10 @@ export class QueryHistoryManager {
   constructor(
     ctx: ExtensionContext,
     private queryHistoryConfigListener: QueryHistoryConfig,
-    selectedCallback?: (item: CompletedQuery) => Promise<void>
+    private selectedCallback: (item: CompletedQuery) => Promise<void>,
+    private doCompareCallback: (from: CompletedQuery, to: CompletedQuery) => Promise<void>,
   ) {
     this.ctx = ctx;
```

Comment:
any reason not to replace `this.ctx = ctx` with an argument list field declaration also?

Reply:
Makes sense. Must have missed that one.

Reply:
Actually...don't need it to be a class property at all. removed the field.